### PR TITLE
data: piaseckijulian

### PR DESCRIPTION
**Pull request by piaseckijulian**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7059"></a>

### CODE REVIEW

File: piaseckijulian.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Julian Piasecki",
+    "avatar": "https://github.com/piaseckijulian.png",
```
Comment (khushalarora) [inline piaseckijulian.json:3]: You can remove this line, it is not required anymore :+1: 
```Suggestion
```

File: piaseckijulian.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "displayStatsPublic": true,
```
Comment (khushalarora) [inline piaseckijulian.json:4]: You can remove this line, it is not required anymore :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @piaseckijulian, thank you for contributing :tada: 
Please rename your file and Add **`data/`** in the beginning [Here](https://github.com/piaseckijulian/LinkFree/edit/piaseckijulian-patch-1/piaseckijulian.json).
Your file name will look like **`data/piaseckijulian.json`**

**Comment by SaraJaoude** [comment]
@piaseckijulian it appears that you have made these changes in a new Pull Request: https://github.com/EddieHubCommunity/LinkFree/pull/7061 so I will be closing this Pull Request. 

In future please close Pull Requests you no longer require - this helps maintainers manage the project.

**Pull request closed without merging**